packages/dm-core-plugins/src/view_selector/Sidebar.tsx

```diff
@@ -12,24 +12,20 @@ export const Sidebar = (props: {
 }): React.ReactElement => {
   const { selectedViewId, setSelectedViewId, viewSelectorItems, addView } =
     props
-
   return (
     <SideBar open style={{ height: 'auto' }}>
       <SideBar.Content>
         {viewSelectorItems.map((viewItem: TItemData) => {
           // subItem's will be rendered inside other items. Don't add them here
           if (viewItem.isSubItem) return null
-
           return (
             <div key={viewItem.viewId}>
               {viewItem.subItems ? (
                 <SideBar.Accordion
                   key={viewItem.viewId}
                   icon={
-                    viewItem.viewConfig?.eds_icon
-                      ? EdsIcons[
-                          viewItem.viewConfig.eds_icon as keyof typeof EdsIcons
-                        ]
+                    viewItem.eds_icon
```

Comment:
This does not keep backwards compatibility

Reply:
Fixed